js/main.js

```diff
@@ -0,0 +1,391 @@
+	    $('html,body').animate({scrollTop: $('#' + id).offset().top},'fast');
+	}
+    
+    // info for restrictions
+    var restrictions = {
+/*
+		'Dairy-Free': {
+			name: 'Dairy-Free',
+			icon: '<i class="fas fa-cheese-swiss"></i>',
+			recipe: '&health=dairy-free',
+			menu: '&intolerances=dairy'
+		},
+		'Egg-Free': {
+			name: 'Egg-Free',
+			icon: '<i class="fas fa-egg"></i>',
+			recipe: '&health=egg-free',
+			menu: '&intolerances=egg'
+		},
+		'Gluten-Free': {
+			name: 'Gluten-Free',
+			icon: '<i class="fas fa-bread-loaf"></i>',
+			recipe: '&health=gluten-free',
+			menu: '&intolerances=gluten'
+		},
+		'Keto': {
+			name: 'Keto',
+			icon: '<i class="fas fa-meat"></i>',
+			recipe: '&health=keto-friendly',
+			menu: '&diet=ketogenic'
+		},
+		'Paleo': {
+			name: 'Paleo',
+			icon: '<i class="fas fa-apple-alt"></i>',
+			recipe: '&health=paleo',
+			menu: '&diet=paleo'
+		},
+		'Pescatarian': {
+			name: 'Pescatarian',
+			icon: '<i class="fas fa-fish-cooked"></i>',
+			recipe: '&health=pescatarian',
+			menu: '&diet=pescetarian'
+		},
+		'Pork-Free': {
+			name: 'Pork-Free',
+			icon: '<i class="fas fa-pig"></i>',
+			recipe: '&health=pork-free',
+			menu: '&diet=vegetarian'
+		},
+		'Red Meat-Free': {
+			name: 'Red Meat-Free',
+			icon: '<i class="fas fa-steak"></i>',
+			recipe: '&health=red-meat-free',
+			menu: '&diet=vegetarian'
+		},
+		'Shellfish-Free': {
+			name: 'Shellfish-Free',
+			icon: '<i class="fas fa-fish"></i>',
+			recipe: '&health=shellfish-free',
+			menu: '&intolerances=shellfish'
+		},
+		'Soy-Free': {
+			name: 'Soy-Free',
+			icon: '<i class="fas fa-seedling"></i>',
+			recipe: '&health=soy-free',
+			menu: '&intolerances=soy'
+		},
+		'Wheat-Free': {
+			name: 'Wheat-Free',
+			icon: '<i class="fas fa-wheat"></i>',
+			recipe: '&health=wheat-free',
+			menu: '&intolerances=wheat'
+		},
+*/
+		'Peanut-Free': {
+			name: 'Peanut-Free',
+			icon: '<i class="fas fa-acorn"></i>',
+			recipe: '&health=peanut-free',
+			menu: '&intolerances=peanut'
+		},
+		'Tree-Nut-Free': {
+			name: 'Tree-Nut-Free',
+			icon: '<i class="fas fa-tree-alt"></i>',
+			recipe: '&health=tree-nut-free',
+			menu: '&intolerances=tree-nut'
+		},
+		'Vegan': {
+			name: 'Vegan',
+			icon: '<i class="fas fa-carrot"></i>',
+			recipe: '&health=vegan',
+			menu: '&diet=vegan'
+		},
+		'Vegetarian': {
+			name: 'Vegetarian',
+			icon: '<i class="fas fa-salad"></i>',
+			recipe: '&health=vegetarian',
+			menu: '&diet=vegetarian'
+		}
+    };
+    
+    // create content section   
+	var content = $('<section>').attr('id', 'content');
+	var contentContainer = $('<div>').addClass('container');
+	
+	// create map container
+	var map = $('<section>').attr('id', 'map');	
+    
+    // create footer   
+	var footer = $('<footer>').attr('id', 'container');
+	footer.html('<div class="row"><div class="col s12"><h2>Didn’t find what you’re looking for?</h2><a href="#hero" class="btn-large deep-orange lighten-2">Search Again</a></div></div>');
```

Comment:
this is fine, but for html that is this long, i may have added it to the page and then show/hide it with jquery